Makefile

```
@@ -46,13 +28,25 @@ build-dev:
 
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 
-run-dev:
-	docker run --interactive --tty --volume=$(dir $(mkfile_path))openslides_backend:/srv/code/openslides_backend \
+dev_args=--interactive --tty \
+	--volume=$(dir $(mkfile_path))openslides_backend:/srv/code/openslides_backend \
+	--volume=$(dir $(mkfile_path))tests:/srv/code/tests \
 	--network=host --rm openslides-backend-dev
 
-run-dev-interactive:
-	docker run --interactive --tty --volume=$(dir $(mkfile_path))openslides_backend:/srv/code/openslides_backend \
-	--network=host --rm openslides-backend-dev sh
+run-dev:
+	docker run $(dev_args)
+
+run-dev-interactive run-bash:
+	docker run $(dev_args) sh
+
+run-cleanup:
+	docker run $(dev_args) sh cleanup.sh
+
+run-tests:
+	docker run $(dev_args) sh OPENSLIDES_BACKEND_RUN_ALL_TESTS=1 pytest
```

sh: can't open 'OPENSLIDES_BACKEND_RUN_ALL_TESTS=1': No such file or directory

---

```suggestion
	docker run $(dev_args) sh "OPENSLIDES_BACKEND_RUN_ALL_TESTS=1 pytest"
```

Correct quoting does the trick.

---

Another thing: I got an INTERNALERROR `AssertionError: /home/finn/OpenSlides/OpenSlides/openslides-backend/tests/integration/test_database.py` inside the container -> you have to remove all `__pycache__` folders and `*.pyc` files. Those compiled artifacts were mounted inside the container and confused pytest.

---

> you have to remove all __pycache__ folders and *.pyc files.

Ahh, that's what I was missing, thanks.

> docker run $(dev_args) sh "OPENSLIDES_BACKEND_RUN_ALL_TESTS=1 pytest"

Didn't work for me, the `-c` switch did the trick. But after pytest didn't work, I didn't bother to look at that.